### ISSUE TITLE
SetValue of type Int returns false with Kwikset lock 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenZWave/open-zwave#404]
```
What steps will reproduce the problem?
1. Attempt to set int values "Timeout Minutes" or "Timeout Seconds" to Kwikset 
lock always return False and not sent to Node. 
According to zwcfg*.xml both are NOT read only:

Value type="int" genre="system" instance="1" index="3" label="Timeout Minutes" 
units="Mins" read_only="false" write_only="false" verify_changes="false" 
poll_intensity="0" min="-2147483648" max="2147483647" value="254" />
                <Value type="int" genre="system" instance="1" index="4" label="Timeout Seconds" units="Secs" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="254" /> 

Sending string value to Kwikset node works. 
Sending int value to other Non Kwikset nodes work.

2.
3.

What is the expected output? What do you see instead?
New value to set in node

What version of the product are you using? On what operating system?
1.2.919
Windows 7

What branch of the code were you using? (if you just followed the
instructions on the source checkout page, then say trunk.. if you checked
out a branch, please let us know what branch it was)
Trunk

Please include the file OZW_Log.txt from your program. Make sure the file
zwcfg*.xml is NOT present.
I can send later if needed. 

Please provide any additional information below.


```

Original issue reported on code.google.com by `TJTrolin...@gmail.com` on 28 Nov 2014 at 4:23

**Comment by minorai** [comment]
This pull request should fix the problem with always "FALSE" response from the setValue function
https://github.com/OpenZWave/open-zwave/pull/1005

**Comment by Fishwaldo** [comment]
Closing as fixed in #1005
